### ISSUE TITLE
Disable UpdateChecking Config Useless

### ISSUE THREAD

**Issue by TataTawa** [Mithion/ArsMagica2#1532]
I play arsmagica2 without internet. So every MC start I must wait for 30s to get Java Connection error to run the game. Disable UpdateChecking Config  is Useless, I notice that in mod initialising phase 3 i still have to  wait 30 seconds every time.

**Comment by ThatOneTraitor** [comment]
are you sure the config is set to false?

**Comment by jjtParadox** [comment]
Any update on this issue?

(@Kolatra)

**Comment by TataTawa** [comment]
yes, i find the problem again. Without Internet In PostInitializing phase the game freezes forever, showing loading ars magica.
there is nothing in log. I guess it should be some internet requirment over and over.
Will there be a way to use ars magica without internet?

**Comment by jjtParadox** [comment]
I'll look into it when I find time :)

**Comment by TataTawa** [comment]
thankyou for your rapid reply :) Sorry for not being a native English speaker 0-0
